### PR TITLE
Fix: Fixed an issue where the name column had a max width

### LINKED ISSUE #10230: Bug: File name column has a max width
### Description

I can't stretch the file name column past a certain width This is very confusing to me. Uninstalling and reinstalling is still the same.

### Steps To Reproduce
1. Open folder in the details layout
2. Try resizing the name column as wide as possible

### Steps to fix
- Remove the max width

### Files Version
v3.3.12

### Windows Version

Win 10 21H2

### PR DESCRIPTION
**Pull request by ferrariofilippo**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Closes #10230 

**Validation**
How did you test these changes?
- [x] Built and ran the app

### CODE REVIEW

File: src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml [hunk 1]
```diff
@@ -299,8 +299,7 @@
 									<ColumnDefinition Width="Auto" />
 									<ColumnDefinition
 										x:Name="Column2"
-										Width="{x:Bind ColumnsViewModel.NameColumn.Length, Mode=OneWay}"
-										MaxWidth="{x:Bind ColumnsViewModel.NameColumn.MaxLength, Mode=OneWay}" />
```
Comment (yaichenbaum) [inline src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml:303]: Should `MaxLength` be removed from the ViewModel?

Comment (ferrariofilippo) [reply src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml:303]: Personally, I would remove this constraint. I don't think it's a problem if the user has very wide columns.

Comment (yaichenbaum) [reply src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml:303]: Most columns should have this constraint, NameColumn is the exception. Is it possible to set the default for the NameColumn MaxLength to *?

Comment (ferrariofilippo) [reply src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml:303]: We can set `double.MaxValue` as default value

Comment (yaichenbaum) [reply src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml:303]: We don't actually have to bind to it in the xaml but I want the c# value to reflect it.

Comment (ferrariofilippo) [reply src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml:303]: What about something like this:

        private ColumnViewModel nameColumn = new ColumnViewModel()
        {
            NormalMaxLength = double.PositiveInfinity
        };

Comment (yaichenbaum) [reply src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml:303]: Maybe just set the max width to 1000 😂

**Review by yaichenbaum — APPROVED** [review]
LGTM

**Pull request merged**